### PR TITLE
Update Readme

### LINKED ISSUE #2: Update Readme (setup instructions)
The setup instructions for development environment need to be completed. The user needs to copy the cookie from auther.

### PR DESCRIPTION
**Pull request by Ashwin-op**
I have made changes to **Readme.md** to include the instructions on what to do if developers are not able to sign in locally.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -77,21 +77,21 @@
       }
```
Comment (arpansac) [inline package-lock.json]: Please remove these files from this commit/PR. We'll update once a major version change is needed.

**Review by arpansac — CHANGES_REQUESTED** [review]
All well, just remove the files package and package-lock.json from this PR and we'll be good to go

File: package-lock.json (hunk 1 above)
Comment (Ashwin-op) [reply package-lock.json]: Should I copy the package.json from the original repo(`commudle/commudle-ng`) and make a commit in mine and again send a PR?

**Comment by arpansac** [comment]
You'll need to use the same versions of the libraries which are used in the
master branch. Try with taking a fresh pull and doing npm install.

**Comment by Ashwin-op** [comment]
I think it's ok now?

**Review by arpansac — APPROVED** [review]

**Comment by arpansac** [comment]
closes #2

**Pull request merged**